src/app/demo-metronic/demo-metronic.component.ts

```diff
@@ -1,8 +1,8 @@
-import { Component, OnInit } from '@angular/core';
-import { AbstractControl, FormControl } from '@angular/forms';
-import { of } from 'rxjs';
+import {Component, OnInit} from '@angular/core';
```

Comment:
Can you please setup your IDE so it formats imports using {_XXX_} (like we all do) instead of {XX}?

Reply:
Sorry, it is resolved now